tools/transform.cpp

```diff
@@ -1880,6 +1880,7 @@ void Transform::print(ostream &os, const TransformPrintOpts &opt) const {
     os << "=>\n";
     tgt.print(os, opt.print_fn_header);
   }
+  std::flush(os);
```

Comment:
it's a little tricky to flush in arm-tv, since the Transform is created then printed all within alive2 code. i made this change to flush the output before it starts the smt verification. in case of timeout, backend-tv might be killed before it flushes. 

it might be easiest to just undo this change - i can't see an easy way to mimic this from arm-tv. what do you think? it is also a very minor one-off change, so should not cause many conflicts if left in.